examples/speech_to_text_v1.py

```diff
@@ -21,3 +23,35 @@
                 timestamps=True,
                 word_confidence=True),
             indent=2))
+
+# Example using websockets
+class MyRecognizeCallback(RecognizeAbstractCallback):
+    def __init__(self):
+        pass
+
+    def on_transcription(self, transcript):
+        print(transcript)
+
+    def on_connected(self):
+        print('Connection was successful')
```

Comment:
use the method name as the string
`print('on_connected()')`

Reply:
This is just an example code. A user can put in any statements he/she likes ;)